ark/segmentation/marker_quantification_test.py

```diff
@@ -161,6 +161,9 @@ def test_assign_multi_compartment_features():
         sample_marker_counts, regionprops_multi_comp
     )
 
+    # assert we added nc_ratio as a features key
+    assert 'nc_ratio' in sample_marker_counts.features.values
+
```

Comment:
Did you check that before this change you could see the warning, and after the warning no longer shows up?

Reply:
Yes, not seeing this warning pop up anymore 👍.